proto/rogue.proto

```diff
@@ -64,25 +64,51 @@ message RogueTalents {
 	bool premeditation = 51;
 }
 
+enum RogueRune {
+	RogueRuneNone = 0;
+	RuneDeadlyBrew = 399965;
+	RuneJustAFleshWound = 400014;
+	RuneQuickDraw = 398196;
+	RuneSlaughterFromTheShadows = 424925;
+	RuneBetweenTheEyes = 400009;
+	RuneBladeDance = 400012;
+	RuneEnvenom = 399963;
+	RuneMainGauche = 424919;
+	RuneMutilate = 399956;
+	RuneSaberSlash = 424785;
+	RuneShadowstrike = 399985;
+	RuneShiv = 424799;
+	RuneMasterOfSubtlety = 425096;
+	RuneWaylay = 408700;
+	RuneRollingWithThePunches = 400093;
+	RuneShadowstep = 400101;
+	RuneShurikenToss = 400096;
+	RunePoisonedKnife = 425102;
+}
+
+message RogueOptions {
+	enum PoisonImbue {
+		NoPoison = 0;
+		InstantPoison = 1;
+		DeadlyPoison = 2;
+		WoundPoison = 3;
+	}
+
+	PoisonImbue mh_imbue = 1;
+	PoisonImbue oh_imbue = 2;
```

Comment:
🔨 We talked about moving these into the standard imbue enum options and restricting them to Rogue only in the UI like we did with Shaman imbues